server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4760,15 +4762,14 @@ protected String validateUserData(String userData, HTTPMethod httpmethod) {
     }
 
     @Override
-    @ActionEvent(eventType = EventTypes.EVENT_VM_CREATE, eventDescription = "starting Vm", async = true)
+    @ActionEvent(eventType = EventTypes.EVENT_VM_START, eventDescription = "starting Vm", async = true)
```

Comment:
This seems correct however it may impact on usage, opinions? @DaanHoogland @rohityadavcloud @harikrishna-patnala @weizhouapache

Reply:
I can revert if its breaking something but I saw this and made change.

Reply:
Action Events don't impact usage. It'll impact listEvents only